accounting/localizations/getting_started.rst

```diff
@@ -0,0 +1,491 @@
+        /!\ This option is only available if your module depends of l10n_multilang.
+        You must provide the language codes separated by ';', e.g. eval="'en_US;ar_EG;ar_SY'". -->
+        <field name="spoken_languages" eval="..."/>
+    </record>
+
+For example, let's take a look to the Belgium chart of accounts.
+
+.. code-block:: xml
+
+    <record id="l10nbe_chart_template" model="account.chart.template">
+        <field name="name">Belgian PCMN</field>
+        <field name="currency_id" ref="base.EUR"/>
+        <field name="bank_account_code_prefix">550</field>
+        <field name="cash_account_code_prefix">570</field>
+        <field name="spoken_languages" eval="'nl_BE'"/>
+    </record>
+
+Now that the chart of accounts is created, we can focus on the creation of the templates.
+As said previously, each record must reference this record through the ``chart_template_id`` field.
+If not, the template will be ignored. The following sections show in details how to create these templates.
+
+Adding a new account to my Chart of Accounts
+############################################
+
+It's time to create our accounts. It consists to creating records of ``account.account.template`` type.
+Each ``account.account.template`` is able to create an ``account.account`` for each company.
+
+.. code-block:: xml
+
+    <record id="..." model="account.account.template">
+        <!-- [Required] Specify the name to display for this account. -->
+        <field name="name">...</field>
+
+        <!-- [Required] Specify a code. -->
+        <field name="code">...</field>
+
+        <!-- [Required] Specify a type. -->
+        <field name="user_type_id">...</field>
+
+        <!-- [Required] Set the CoA owning this account. -->
+        <field name="chart_template_id" ref="..."/>
+
+        <!-- [Optional] Specify a secondary currency for each account.move.line linked to this account. -->
+        <field name="currency_id" ref="..."/>
+
+        <!-- [Optional] Boolean to allow the user to reconcile entries in this account. True by default. -->
+        <field name="reconcile" eval="..."/>
+
+        <!-- [Optional] Specify a group for this account. -->
+        <field name="group_id" ref="...">
+
+        <!-- [Optional] Specify some tags. -->
+        <field name="tag_ids" eval="...">
+    </record>
+
+Some of the described fields above deserve a bit more explanation.
+
+The ``user_type_id`` field requires a value of type ``account.account.type``.
+Although some additional types could be created in a localization module, we encourage the usage of the existing types in the `account/data/data_account_type.xml <https://github.com/odoo/odoo/blob/11.0/addons/account/data/data_account_type.xml>`_ file.
+The usage of these generic types ensures the generic reports working correctly in addition to those that you could create in your localization module.
+
+.. warning::
+
+    * Avoid the usage of liquidity ``account.account.type``.
```

Comment:
since the bank/cash accounts will be created at the installation of the localization and directly linked to an account.journal